scss/layout/_layouts.scss

```diff
@@ -11,6 +11,11 @@
     width: 46%;
 }
 
+.l-space {
```

Comment:
Rename to something icon related. Make sure to update name in the footer.php file as well.